src/Parallelization.php

```diff
@@ -359,19 +359,17 @@ function () use ($input) {
                 sprintf('The bin/console file could not be found at %s', getcwd()))
             ;
 
-            $commandTemplate = implode(
-                ' ',
-                array_merge(
-                    array_filter([
-                        self::detectPhpExecutable(),
-                        $consolePath,
-                        $this->getName(),
-                        implode(' ', array_slice($input->getArguments(), 1)),
-                        '--child',
-                    ]),
-                    $this->serializeInputOptions($input, ['child', 'processes'])
-                )
+            $commandTemplate = array_merge(
+                array_filter([
+                    self::detectPhpExecutable(),
+                    $consolePath,
+                    $this->getName(),
+                    implode(' ', array_slice($input->getArguments(), 1)),
+                    '--child',
+                ]),
+                $this->serializeInputOptions($input, ['child', 'processes'])
```

Comment:
it is necessary to remove white spaces at the beginning of input options, otherwise shellescaping will do strange things. 
So modify the method `serializeInputOptions` as follows: 

```php
    private function serializeInputOptions(InputInterface $input, array $blackListParams): array
    {
        $options = array_diff_key(
            array_filter($input->getOptions()),
            array_fill_keys($blackListParams, '')
        );

        $preparedOptionList = [];
        foreach ($options as $name => $value) {
            $definition = $this->getDefinition();
            $option = $definition->getOption($name);

            $optionString = '';
            if (!$option->acceptValue()) {
                $optionString .= '--'.$name;
            } elseif ($option->isArray()) {
                foreach ($value as $arrayValue) {
                    $optionString .= '--'.$name.'='.$arrayValue;
                }
            } else {
                $optionString .= '--'.$name.'='.$value;
            }

            $preparedOptionList[] = $optionString;
        }

        return $preparedOptionList;
    }
```